src/frontend/src/flows/recovery/inputSeedPhrase.ts

```diff
@@ -69,3 +98,12 @@ const dropLeadingUserNumber = (s: string): string => {
     return s;
   }
 };
+
+const leadingUserNumber = (s: string): bigint | null => {
+  const i = s.indexOf(" ");
```

Comment:
Do you want to trim() first, so that leading whitespace does not matter?  Something like this maybe?
```
let [num_str] = s.trim().split(" ", 1) // get first word.
if ("" !== num_str) {
  return parseUserNumber(num_str);
...
```
I know I sometimes copy and paste passphrases and catch an additional whitespace at the beginning and it can be really confusing if the password is "incorrect" but I can see no problem.  With passwords one is rather more attuned to every character mattering, so is more careful.

Reply:
Ok I've addressed this a bit differently. The code here stays the same (mirroring `dropLeadingUserNumber` above) but I trim the value before calling `leadingUserNumber`. Thanks!